### PR TITLE
UPBGE: Implement Dynamic Texture LOD Bias

### PR DESCRIPTION
**Pull request by DCubix**
It allows the user to specify any value for texture lod bias, such as roughness maps, ramps, and a lot more.
It's possible to use the same texture multiple times!

Test file: http://pasteall.org/blend/index.php?id=46353

### CODE REVIEW

**Comment by DCubix** [comment]
Using that I was able to create a PBR node setup.
pbr

.blend: http://pasteall.org/blend/index.php?id=46359

File: source/blender/makesrna/intern/rna_nodetree.c [hunk 1]
```diff
@@ -136,11 +136,13 @@ EnumPropertyItem rna_enum_node_math_items[] = {
...
-	{3, "DOT_PRODUCT",   0, "Dot Product",   ""},
-	{4, "CROSS_PRODUCT", 0, "Cross Product", ""},
-	{5, "NORMALIZE",     0, "Normalize",     ""},
+	{1, "MULTIPLY",      0, "Multiply",      ""},
+	{2, "SUBTRACT",      0, "Subtract",      ""},
+	{3, "AVERAGE",       0, "Average",       ""},
+	{4, "DOT_PRODUCT",   0, "Dot Product",   ""},
+	{5, "CROSS_PRODUCT", 0, "Cross Product", ""},
+	{6, "REFLECT",       0, "Reflect",       ""},
+	{7, "NORMALIZE",     0, "Normalize",     ""},
```
Comment (panzergame) [inline source/blender/makesrna/intern/rna_nodetree.c]: There is no compatibility issue with the index renaming ?

Comment (DCubix) [inline source/blender/makesrna/intern/rna_nodetree.c]: Yes there is! 😮 I'll fix it!

**Comment by panzergame** [comment]
Merged in master.

**Pull request closed without merging**